go/obscuronode/nodecommon/types.go

```diff
@@ -15,9 +15,9 @@ import (
 
 type (
 	StateRoot             = common.Hash
-	EncryptedTx           []byte
 	L2Tx                  = types.Transaction
-	EncryptedTransactions []EncryptedTx
+	EncryptedTx           []byte
```

Comment:
related to the comment about the different types of encryption. Let's add a comment here as well